packages/quantic/force-app/main/default/lwc/quanticSort/quanticSort.html

```diff
@@ -4,6 +4,7 @@
     </c-quantic-component-error>
   </template>
   <template lwc:else>
+    <slot><!-- Custom sort options --></slot>
```

Comment:
When using slots like this, you are able to provide the slot that will be used by default.

Since in our the case the default situation is to have `relevancy`, `newest` and `oldest` options, it would make send sense to represent this default options with the QuanticSortOption component and pass them as default values in this slot to avoid having to check in the JS if custom options are used or not.

you will have something like this:
```
<slot>
  <c-quantic-sort-option label="relevancy" value="..." criterion="..." ></quantic-sort-option>
  <c-quantic-sort-option label="newest" value="..." criterion="..." ></quantic-sort-option>
  <c-quantic-sort-option label="oldest" value="..." criterion="..." ></quantic-sort-option>
</slot>
```

Reply:
I think I would prefer to leave the slots for custom options and leave the current logic with the buildOptions() method for the default options and leave the lightning-combobox with the options prop.

rather than making multiple getters for values and criterions for the default options.